clientapi/routing/sendtyping.go

```diff
@@ -46,46 +45,19 @@ func SendTyping(
 	}
 
 	// Verify that the user is a member of this room
-	tuple := gomatrixserverlib.StateKeyTuple{
-		EventType: gomatrixserverlib.MRoomMember,
-		StateKey:  userID,
-	}
-	var res currentstateAPI.QueryCurrentStateResponse
-	err := stateAPI.QueryCurrentState(req.Context(), &currentstateAPI.QueryCurrentStateRequest{
-		RoomID:      roomID,
-		StateTuples: []gomatrixserverlib.StateKeyTuple{tuple},
-	}, &res)
-	if err != nil {
-		util.GetLogger(req.Context()).WithError(err).Error("QueryCurrentState failed")
-		return jsonerror.InternalServerError()
-	}
-	ev := res.StateEvents[tuple]
-	if ev == nil {
-		return util.JSONResponse{
-			Code: http.StatusForbidden,
-			JSON: jsonerror.Forbidden("User not in this room"),
```

Comment:
The message will now be `user does not belong to room` (see `checkMemberInRoom`)